src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -270,6 +271,26 @@ log_metadata_difference(Ledger::pointer builtLedger, Ledger::pointer validLedger
     }
 }
 
+//------------------------------------------------------------------------------
+
+// Return list of leaves sorted by key
+static
+std::vector<std::shared_ptr<
+    SHAMapItem const>>
+leaves (SHAMap const& sm)
+{
+    std::vector<std::shared_ptr<
+        SHAMapItem const>> v;
+    for (auto const& item : sm)
+        v.push_back(item);
+    std::sort(v.begin(), v.end(),
+        [](std::shared_ptr<SHAMapItem const> const& lhs,
```

Comment:
And a few lines later, you don't break this same classname in a line 56 characters long!